actors/common/src/main/java/org/sunbird/learner/actors/BackgroundJobManager.java

```diff
@@ -113,6 +114,8 @@ public void onReceive(Request request) throws Throwable {
       insertUserNotesToEs(request);
     } else if (operation.equalsIgnoreCase(ActorOperations.UPDATE_USER_NOTES_ES.getValue())) {
       updateUserNotesToEs(request);
+    } else if (operation.equalsIgnoreCase(ActorOperations.MERGE_USERS_TO_ELASTIC.getValue())) {
```

Comment:
MERGE_USERS_TO_ELASTIC to MERGE_USER_TO_ELASTIC

Reply:
Done